### DISCUSSION TITLE
Incorrect formatting of Markdown source in code span

### DISCUSSION THREAD

**Discussion by 0cjs** [community/community#33333] [Other Feature Feedback, Questions, & Ideas]
When rendering a Markdown file from a repo in the web interface, GitHub normally handles code spans that cross lines without problems. For example,
```
- This point suggests you `echo
  hello` to blah blah blah.
```
will render "echo hello" as a code span: `echo hello`.

However, it seems that if the continuation line starts with the shell redirect character, `>`, GitHub interprets this as the code span no longer being a code span and instead the first character of the continuation line is interpreted as markup for a block quote. E.g.:
```
- This point suggests you `echo
  >somefile hello` to blah blah blah
  ```
  will render "echo >somefile hello" not as `echo >somefile hello` but as "&#96;echo" followed by a block-quoted "somefile hello&#96;" on the next line.

You can see an example of this [here](https://github.com/0cjs/sedoc/blob/313b7bf216f44ebf9bc06435931291ff5bffc0db/docker/image.md#build-cache), in the sixth paragraph (second indented bullet point) after the "Build Cache" heading.

**Comment by 0cjs** [comment]
I notice that this has been sitting around for three weeks now without a response. Is there some other place I should be posting this?

**Reply by queenofcorgis** [reply]
Thanks for posting in the GitHub Community, @0cjs ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by 0cjs** [reply]
Ok. I think this was originally in the "General" discussion forum, but when I saw an area labeled "Issues" I assumed that was for reporting issues with GitHub's software.

Really, this should be filed as a bug report in an issues system rather than posted in a discussion forum, should it not? (This looks more like a forum for users to help other users, which obviously is not too useful for "GitHub actually doesn't work.") Is there no place to report bugs in GitHub? It seems rather ironic that you provide public bug tracking for so many software projects, yet do not have that for your own software.

**Comment by 0cjs** [comment]
It's been over a month since I reported this bug. Have GitHub's developers even been informed about it?
